proxy/server_test.go

```diff
@@ -63,7 +63,7 @@ func (f fakeListener) dialer() func(addr string, c *tls.Config) (net.Conn, error
 func setupTestServer(tb testing.TB, cacheSize int, responder func(q string) string) (exchanger func(logPrefix string), cleanup func()) {
 	tb.Helper()
 	const (
-		raddr    = "gopher.empijei:853"
+		raddr    = "gopher.empijei:53"
```

Comment:
Shouldn't it have :853 as it is a DNS-over-TLS upstream?

Reply:
Done.